pod/perlguts.pod

```diff
@@ -117,6 +117,11 @@ the SV has the undef value.
     SV *sv = newSV(10);  /* 10 (+1) bytes of uninitialised storage
                           * allocated */
 
+Or you can create an SV that is initialized to an empty string, by using
+L<C<newSVpvz(len)>|perlapi/newSVpvz>.  Perl will not have to ask for
+more space for it from the operating system until and unless the string
```

Comment:
I'm not sure if we need to be this specific about "from the operating system", unless there's something more specific going on here. Maybe just some words about "Perl will not have to reallocate space unless and until the string ..."